apps/wallet-mobile/src/features/Swap/useCases/CreateOrder/EditSlippageScreen.tsx

```diff
@@ -72,7 +72,7 @@ export const EditSlippageScreen = () => {
   const onSubmit = () => {
     const slippage = selectedChoice.label === 'Manual' ? parseNumber(inputValue, numberLocale) : selectedChoice.value
     track.swapSlippageChanged({slippage_tolerance: slippage})
-    slippageChanged(slippage)
+    swapForm.dispatch({type: 'SlippageInputChanged', value: slippage})
```

Comment:
- [ ] `.dispatch` leaks implementation, wrap form actions in helper functions

Reply:
@stackchain not sure I agree here. It doesn't "leak" because there's no module, package or isolation anyway.
SwapProvider is part of mobile, and that's where it's used.
The dispatches are typed, so both `type` and `value` get autocomplete, you don't have to go into the provider to look up what's supposed to go in.
Creating an additional 20 named functions that basically wrap a dispatch seems just like bloat.

Reply:
you don't need a module to leak an implementation, is like creating args for a signature with `nameArray`, `arrayOfName`, the implementation should be exposed, even internally.

Reply:
`xxxForm.update` looks like a good API

Reply:
I'd rather wrap it, cuz simplifies for those consuming, abstract the payload away.

Reply:
renamed to update